nservicebus/scheduling/index_content_core_[8,).partial.md

```diff
@@ -0,0 +1,7 @@
+Executing code on various intervals is a common need when building enterprise systems. NServiceBus provides the following uptions for scheduling:
```

Comment:
```suggestion
Executing code at various intervals is a common need when building enterprise systems. Particular Software proposes the following options for scheduling:
```

Reply:
I'm not sure we should use the term "NServiceBus provides", because we apart from sagas and the samples, the rest is non-nsb